telegram/ext/updater.py

```diff
@@ -251,11 +255,54 @@ def start_polling(self,
         Returns:
             :obj:`Queue`: The update queue that can be filled from the main thread.
 
+        Raises:
+            ValueError: If :attr:`clean` is set as :obj:`datetime.timedelta` and is < 1 second.
+
         """
         with self.__lock:
             if not self.running:
                 self.running = True
 
+                self.logger.debug('clean:               "%s"', type(clean))
+                self.logger.debug('clean:               "%s"', clean)
+                self.logger.debug('now:                 "%s"', datetime.now(tz=timezone.utc))
+
+                if isinstance(clean, timedelta):
+                    if clean.total_seconds() < 1:
+                        raise ValueError('Clean as timedelta needs to be >= 1 second')
+                    else:
+                        a=(datetime.now(tz=timezone.utc) - clean)
```

Comment:
Please use descriptive variable names. Also this is computed twice (Line 279)

Reply:
part of cleanup once y'all are happy with the code itself. I'll wrap it in a comment to make it clearer this is just for debugging while developing